src/packet/all_packets.js

```diff
@@ -4,75 +4,73 @@
  * @module packet/all_packets
  */
 
-import * as packets from './all_packets.js'; // re-import module to parse packets from tag
-
 export {
-  /** @see module:packet.Compressed */
-  default as Compressed
-} from './compressed.js';
+  /** @see CompressedDataPacket */
```

Comment:
Those long name are kinda ugly in my opinion, may be just use CompressedData or Compressed?

Reply:
["Compressed Data Packet"](https://tools.ietf.org/html/rfc4880#section-5.6) is the official name of the packet, this way all the names of the classes are consistent with the names of the packets.

Reply:
Yeah, but this is not documentation, it is common to left out common name if classes belong to a same folder. For example Sequoia PGP https://gitlab.com/sequoia-pgp/sequoia/-/blob/master/openpgp/src/packet/compressed_data.rs